projects/HSLink-OB/board/hslinkob/README_zh.md

```
@@ -0,0 +1,109 @@
+
+## 引脚描述
+
+(lab_hpm5301_evklite_board)=
+
+- UART引脚
+  - UART0用于调试控制台串口或一些使用UART的功能测试
+  - UART3用于一些使用UART的功能测试，例如MICROROS_UART，USB_CDC_ACM_UART, MODBUS_RTU等。
+
+| 功能      | 位置   | 备注     |
+| --------- | ------ |  ------ |
+| UART3.TXD | J3[8]  |
```

markdownlint-cli2 (0.17.2)

Table column count
Expected: 3; Actual: 2; Too few cells, row will be missing data

(MD056, table-column-count)